cmd/meshregistrator/local.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
+// check host for services configured via other means and update registration list
+func fetchLocal(ctx context.Context, out chan []ServiceRegistration, servicesDir string) {
```

Comment:
what other means are there for registering in a namespace other than either paasta or puppet (what's a local service)?

Reply:
i could have used "puppet" although services are actually defined in "nerve" folder and i wasn't sure there isn't or won't be another way to register something locally, so i went with most generic name i could think of